### PR TITLE
#1451 Fix calendar icon

### PR DESCRIPTION
**Pull request by AlexBelov**
Add helper to render short localized name of month
screenshot from 2015-08-26 11 28 46

### CODE REVIEW

**Comment by AlexBelov** [comment]
screenshot from 2015-08-26 19 28 39

**Comment by AlexBelov** [comment]
@coorasse Can you check month locale in calendar icon on any old version? It always shows eng month names to me. But solution from this PR (time_helper) fixes it. Seems pretty good-looking, right?

File: app/assets/stylesheets/event.css.scss [hunk 1]
```diff
@@ -54,27 +56,6 @@
   }
 }
 
-// .event_calendar_mini {
```
Comment (coorasse) [inline app/assets/stylesheets/event.css.scss]: remove old code

File: app/assets/stylesheets/event.css.scss [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
 
   .event-month {
     margin: 0;
     background: #eee;
-    padding: 0.5rem 2rem;
+    padding: 0.5rem;
     text-align: center;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
```
Comment (coorasse) [inline app/assets/stylesheets/event.css.scss]: :+1:

File: app/helpers/time_helper.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+module TimeHelper
+  def month_long month
```
Comment (coorasse) [inline app/helpers/time_helper.rb]: use parentheses when defining a method:
https://github.com/bbatsov/ruby-style-guide#method-parens

File: app/views/blogs/_left_panel.html.erb [hunk 4]
```diff
@@ -29,7 +29,7 @@
   <div class="mycont">
     <%@archives.each do |rec|%>
         <div class="blog_comment">
-          <%=link_to "#{t('calendar.monthNames')[rec.month.to_i - 1]} #{rec.year.to_i} (#{rec.posts})", posts_by_year_and_month_blog_path(@blog, month: rec.month.to_i, year: rec.year.to_i) %>
+          <%=link_to "#{month_long(rec.month)} #{rec.year.to_i} (#{rec.posts})", posts_by_year_and_month_blog_path(@blog, month: rec.month.to_i, year: rec.year.to_i) %>
```
Comment (coorasse) [inline app/views/blogs/_left_panel.html.erb]: :+1:

File: app/views/groups/_group_portlet_tag.html.erb [hunk 5]
```diff
@@ -16,20 +15,6 @@
           <div class="fade_out"></div>
         </div>
       </div>
-
```
Comment (coorasse) [inline app/views/groups/_group_portlet_tag.html.erb]: :+1:

**Comment by coorasse** [comment]
please remove the assets related to the calendar:
app/assets/images/calendar.png
app/assets/images/calendar-32.png
app/assets/images/calendar-48.png
app/assets/images/calendar-52.png

File: app/assets/stylesheets/event.css.scss (hunk 1 above)
Comment (AlexBelov) [inline app/assets/stylesheets/event.css.scss]: removed =)

**Comment by AlexBelov** [comment]
Rebased on latest develop

**Comment by coorasse** [comment]
thank you! merged! looks fine to me

**Pull request merged**